tests/integration_test_data_endpoint.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+import re
+
+from syncano.models import Class, DataEndpoint, Object, ResponseTemplate
+from tests.integration_test import InstanceMixin, IntegrationTest
+
+
+class DataEndpointTest(InstanceMixin, IntegrationTest):
+
+    schema = [
+        {
+            'name': 'title',
+            'type': 'string',
+            'order_index': True,
+            'filter_index': True
+        }
+    ]
+
+    template_content = '''
```

Comment:
According out previous talks - yep it's indepedent and already written. But probably there will be no situation that <table> will be removed in CORE in the table_html_list ;)